cypress/support/commands/pageTestFragment/metaData.js

```diff
@@ -53,8 +45,16 @@ Cypress.Commands.add('checkOpenGraphMetadata', (
       // cy.get('meta[name="og:description"]').should('have.attr', 'content', description); // !!! Remove eslint-disabling comment above when un-commenting this test.
     });
 
-    cy.checkMetadataContent('head meta[name="og:locale"]', locale);
-    cy.checkMetadataContent('head meta[name="og:site_name"]', siteName);
+    cy.get('head meta[name="og:locale"]').should(
```

Comment:
The three commands `checkOpenGraphMetadata`, `checkTwitterMetadata`, `checkFacebookMetadata` here are run only in `/pages/articles.js`. These should be run in `/pages/index.js` with any variants in values between pagetypes factored out.

Reply:
There's an incoming PR that reuses them, so I won't do this here. I will double check after that's in though.